farm/evaluation/metrics.py

```diff
@@ -189,11 +206,41 @@ def squad_f1_single(pred, label, pred_idx=0):
     f1 = (2 * precision * recall) / (precision + recall)
     return f1
 
+
+def confidence(preds):
+    conf = 0
+    for pred in preds:
+        conf += pred[0][0].score
+    return conf/len(preds) if len(preds) else 0
+
+
+def metrics_per_bin(preds, labels):
+    pred_bins = [[] for _ in range(10)]
```

Comment:
We can have variable called `num_of_bins=10` and use that for readability purpose.

Reply:
Good idea. I will do that.